features/support/env.rb

```diff
@@ -17,6 +17,10 @@
 
 # Additional require MacOS configuration
 if Maze.config.os == 'macos'
+  # The default macOS Crash Reporter "#{app_name} quit unexpectedly" alert grabs focus which can cause tests to flake.
+  # This option, which appears to have been introduced in macOS 10.11, displays a notification instead of the alert.
+  `defaults write com.apple.CrashReporter UseUNC 1`
```

Comment:
There's also `defaults write com.apple.CrashReporter DialogType none`, if that's clearer.

Reply:
I felt this option was a bit safer than disabling it totally.

It could be confusing to have it completely disabled after running the tests on ones laptop, and miss important crash reports thereafter.

Reply:
Ah, good point 👍